djcelery/models.py

```diff
@@ -182,6 +184,8 @@ class PeriodicTasks(models.Model):
     ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
     last_update = models.DateTimeField(null=False)
 
+    no_changes = False
```

Comment:
That seems reasonable.
But this line is defined on `PeriodicTasks` class. I don't think it is needed here, since the signal is not connected with `sender=PeriodicTasks`. Meanwhile, `IntervalSchedule` doesn't have `no_changes`.